__fixtures__/nodeReleasesData.tsx

```diff
@@ -0,0 +1,121 @@
+import type { NodeReleaseData } from '../types';
+
+export const createNodeReleasesData = () =>
```

Comment:
Do we need to have this many entries inside this fixture? Is this done in order to cover most of the cases of our logic?

Reply:
Yes, I intended to cover up to the entry which after that will be all `End-of-life`.

Maybe we can remove the last 2 entries.

Reply:
Feel free to do it :)